modules/fastcommunity/src/main/java/org/clueminer/fastcommunity/FastCommunity.java

```diff
@@ -1,201 +1,197 @@
-package org.clueminer.fastcommunity;

```

Comment:
This file seems to fail to merge even after the line endings have been changed, i.e. LF is used. Is it possible that the current base version contains wrong line endings?